dockernet/pkill.sh

```diff
@@ -0,0 +1,3 @@
+pkill -f "docker-compose .*simd.* logs" | true
+pkill -f "/bin/bash.*create_logs.sh" | true
```

Comment:
Can we move this file to dockernet?

Reply:
I don't really understand, is it already in dockernet?